src/client/actions/MessageReactionRemoveEmoji.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Action = require('./Action');
+const { Events } = require('../../util/Constants');
+
+class MessageReactionRemoveEmoji extends Action {
+  handle(data) {
+    const channel = this.getChannel(data);
+    if (!channel || channel.type === 'voice') return false;
+
+    const message = this.getMessage(data, channel);
+    if (!message) return false;
+
+    const reaction = this.getReaction(data, message);
+    if (!reaction) return false;
+    if (!message.partial) message.reactions.delete(reaction.emoji.id || reaction.emoji.name);
+
+    /**
+     * Emitted when a bot removes a reaction from a cached message.
```

Comment:
It does more than remove a reaction right? It should detail that it removes all reactions of a particular emoji from a message.

Reply:
Well, a reaction is known to both the api and the lib as containing a single emoji and multiple users, so its technically removing a single reaction, i am uncertain of the wording i should use for exactly this reason

Reply:
How can you differentiate singular person clicking on their reaction to remove it, from someone removing whole reaction then? Both are reactions removals, except the latter wasn't possible before.
Personally, i think i would use word `emoji` there, afterall you remove emoji, no matter how many reactions it had.

Also, remember that while yes, for API and library it's reaction, it not always has to be instantly logical for actual users. Just think how many times you have to explain difference between User and GuildMember.